types/react-native/index.d.ts

```diff
@@ -469,6 +472,83 @@ export interface Insets {
     right?: number;
 }
 
+export type PressableStateCallbackType = Readonly<{
+    pressed: boolean;
+}>;
+
+export interface PressableProps extends AccessibilityProps, Omit<ViewProps, 'style' | 'hitSlop'> {
+    /**
+     * Called when a single tap gesture is detected.
+     */
+    onPress?: null | ((event: GestureResponderEvent) => void);
```

Comment:
You can extract ((event: GestureResponderEvent) => void) into separate type.


```typescript
export type GestureResponseCallbackType = (event: GestureResponderEvent) => void;
```

```suggestion
    onPress?: null | GestureResponseCallbackType;
```